Apply max penalty for sub-segment match

t-cordonnier opened the pull request:
During a subsegment search, if some of the subsegments have a penalty, the global segment receives as penalty the highest value

## Pull request type


- Bug fix -> [bug]
- Feature enhancement -> [enhancement]

## Which ticket is resolved?

https://sourceforge.net/p/omegat/bugs/1251/

miurahr commented:
Why don't you use the `maxPanelty` variable to process the entry, but use `0`?
```
processEntry(null, entry, "", NearString.MATCH_SOURCE.TM, false, 0);
```

t-cordonnier commented:
> Why don't you use the `maxPanelty` variable to process the entry, but use `0`?
> 
> ```
> processEntry(null, entry, "", NearString.MATCH_SOURCE.TM, false, 0);
> ```

You are true, I had modified this line in the other PR only, I forgot to do this change when I splitted in two pull requests. I will correct it

t-cordonnier commented:
> > Why don't you use the `maxPanelty` variable to process the entry, but use `0`?
> > ```
> > processEntry(null, entry, "", NearString.MATCH_SOURCE.TM, false, 0);
> > ```
> 
You are true, I had modified this line in the other PR only, I forgot to do this change when I splitted in two pull requests. 

Corrected now
